client/src/app/project/[projectid]/searchComponents/SearchResults.tsx

```diff
@@ -33,8 +84,49 @@ const SearchResults: FC<Props> = ({
       <div className="flex flex-row justify-center items-center pb-4 text-2xl">
         <div className="pr-4 text-lg font-bold">Searched Places</div>
       </div>
-      <div className="flex flex-col items-center h-[calc(100vh-15rem)] overflow-y-auto">
-        {noteData.map((note) => {
+
+      <button onClick={() => {
+        setExpandSort(!expandSort);
+        setExpandFilter(false);
+      }}>Sort</button>
+
+      <button onClick={() => {
+        setExpandFilter(!expandFilter);
+        setExpandSort(false);
+      }}>Filter</button>
+
+      {expandSort ? (
+        <div>
+          <ul>
+            <li><button onClick={() => sortByValue("name")}>by Name</button></li>
+            <li><button onClick={() => sortByValue("date")}>by Date</button></li>
+            <li><button onClick={() => sortByValue("priority")}>by Priority</button></li>
+          </ul>
+        </div>
+      ) : null}
```

Comment:
For both, it looks like expanding the filter/sorts affects the css causing it to overflow. Would you be able to make this into a popup / dialog?